src/Nethermind/Nethermind.Network/Rlpx/NettyHandshakeHandler.cs

```diff
@@ -127,7 +135,7 @@ public override void ExceptionCaught(IChannelHandlerContext context, Exception e
                 }
             }
 
-            base.ExceptionCaught(context, exception);
+            _ = context.DisconnectAsync();
```

Comment:
Don't we need to call base class anymore? Why?

Reply:
Calling base class does nothing. It just pass the exception away to the tail.